README.md

```diff
@@ -6,27 +6,27 @@ This repository contains demo web pages, apps, and sample code used to demonstra
-| **DevTools Console demos** | Multiple web pages to demonstrate various Edge DevTools Console features. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-console) | [Demo](https://microsoftedge.github.io/Demos/devtools-console/) |
-| **Contrast bug fix** | Demo page used in a Microsoft Edge [blog post](https://blogs.windows.com/msedgedev/2021/06/15/improving-contrast-in-microsoft-edge-devtools-a-bugfix-case-study/) about improving contrast in Edge DevTools. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-contrast-bugfix) | [Demo](https://microsoftedge.github.io/Demos/devtools-contrast-bugfix/) |
+| **Edge DevTools VS Code extension sourcemap support** | Simple app to test the sourcemap support for CSS mirroring in the Microsoft Edge DevTools extension for Visual studio code. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/css-mirroring-sourcemaps-demo) | N/A |
+| **TODO app** | A simple To Do app used in the DevTools documentation. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/demo-to-do) | [Demo](https://microsoftedge.github.io/Demos/demo-to-do/) |
+| **Detached Elements** | A chat-like demo application with DOM memory leaks to demonstrate how to use the Detached Elements tool in Microsoft Edge DevTools. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/detached-elements) | [Demo](https://microsoftedge.github.io/Demos/detached-elements/) |
+| **3D View** | A sample web page used to demonstrate how to use the 3D View tool in Microsoft Edge DevTools. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-3d) | [Demo](https://microsoftedge.github.io/Demos/devtools-3d/) |
+| **Animal Shelter** | A sample website with accessibility issues, used to demonstrate how to use the accessibility issues detection features of Microsoft Edge DevTools. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-a11y-testing) | [Demo](https://microsoftedge.github.io/Demos/devtools-a11y-testing/) |
+| **DevTools Console demos** | Multiple web pages to demonstrate various DevTools Console features. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-console) | [Demo](https://microsoftedge.github.io/Demos/devtools-console/) |
+| **Contrast bug fix** | Demo page used in a Microsoft Edge [blog post](https://blogs.windows.com/msedgedev/2021/06/15/improving-contrast-in-microsoft-edge-devtools-a-bugfix-case-study/) about improving contrast in Microsoft Edge DevTools. | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-contrast-bugfix) | [Demo](https://microsoftedge.github.io/Demos/devtools-contrast-bugfix/) |
 | **DevTools CSS getting started docs** | Source code for a demo page used in the [DevTools CSS getting started docs](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/css/). | [Code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-css-get-started) | [Demo](https://microsoftedge.github.io/Demos/devtools-css-get-started/) |
```

Comment:
why is the demo called "... docs"?  Not consistent with purpose of column heading.  Trying clearer naming in the sibling Docs PR now...

Options for "what's the name of the demo":
* Title in page.
* Tab name.
* Directory name.

Reply:
Sure, changing this first column to the docs title makes sense to me.

Reply:
Tentative plan for https://github.com/MicrosoftEdge/Demos#readme in this PR:
* Make column 1 heading "Demo name" instead of "Demo", to be more specific (every column is about the Demo so the column heading "Demo" doesn't differentiate the columns).
* Change column 3 heading from "Code" to "Code directory".
* In each cell of that column, change from "Code" (undifferentiated) to the code directory name, which is needed to navigate the directories at repo and in downloaded copy of repo and practically is the name of the demo re: navigation.
* Move to identical design & Markdown in the Docs PR & this Demos PR.

Docs repo PR's rendered page containing latest design of equivalent table:
https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/clone-demos/microsoft-edge/devtools-guide-chromium/sample-code/sample-code.md#sample-code-for-devtools

Reply:
redesigned in both repos, same Markdown except Docs build will require removing LMC prefix from URLs